src/storage/statistics_cl.c

```diff
@@ -614,7 +606,14 @@ stats_get_ndv_by_query (const MOP class_mop, CLASS_ATTR_NDV * class_attr_ndv, FI
     }
 
   /* create sampling SQL statement */
-  query = "SELECT /*+ SAMPLING_SCAN */ %s FROM [%s]";
+  if (with_fullscan)
```

Comment:
It's a minor thing.
I think it's better for readability to compare `with_fullscan` with `STATS_WITH_FULLSCAN` and `STATS_WITH_SAMPLING`.

Reply:
I fixed it.